lib/rouge/lexers/vcl.rb

```diff
@@ -3,24 +3,31 @@
 
 module Rouge
   module Lexers
-    class Varnish < RegexLexer
-      title 'Varnish'
-      desc 'The Varnish (high-performance web accelerator) configuration language'
+    class VCL < RegexLexer
```

Comment:
Can we revert this to `Varnish`? I'm not opposed to changing this as part of v4.0.0 but this would break any app that had a class which inherited from this class.

Reply:
Sounds fair. Reverted and applied only metadata changes (4b8b1a2dd5c91dff00eb44ef3ead11f8ea0c3330).